compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -214,12 +217,13 @@ private static void addBuiltinImports(PackageID currentModule, Set<PackageID> de
         dependentModuleArray.add(PackageID.TRANSACTION);
     }
 
-    private static boolean isLangModule(PackageID moduleId) {
+    public static boolean isLangModule(PackageID moduleId) {
 
         if (!BALLERINA.equals(moduleId.orgName.value)) {
             return false;
         }
-        return moduleId.name.value.indexOf("lang" + ENCODED_DOT_CHARACTER) == 0 || moduleId.name.equals(Names.JAVA);
+        return moduleId.name.value.indexOf("lang" + ENCODED_DOT_CHARACTER) == 0 ||
```

Comment:
Why don't we use `moduleId.name.value.startsWith("lang" + ENCODED_DOT_CHARACTER)` here, it is better right?